checkers.kv

```diff
@@ -0,0 +1,64 @@
+<TitleMenu>:
+    BoxLayout:
+        orientation: 'vertical'
+        Label:
+            text: 'Checkers'
+        Button:
```

Comment:
As discussed on Discord, as these settings pertain to the instance of the game, they should only be accessible through "Play" being clicked

Reply:
I don't totally follow.  The settings affect how the game is set up.  When the user hits play, don't we want the game to start in the correct mode? Player v cpu / player v player